src/code/addButtons.js

```diff
@@ -1,13 +1,15 @@
-import LinkDialog from "./linkDialog";
-import { MarkerDialog } from "./markerDialog";
-import { OpsDialog } from "./opsDialog";
-import Operation from "./operation";
+import WasabeeMe from "./me";
 import QuickDrawControl from "./quickDrawLayers";
+import WasabeeButtonControl from "./wasabeeButton";
+import opsButtonControl from "./opsButton";
+import NewopButtonControl from "./newopButton";
+import LinkDialogButtonControl from "./linkDialogButton";
+import MarkerButtonControl from "./markerButton";
 
 var Wasabee = window.plugin.Wasabee;
 
 /* This function adds the plugin buttons on the left side of the screen */
-export default function() {
+export default function(selectedOp) {
```

Comment:
I see a lot of changes like this to pass in the selectedOp rather than getting it via the global function, but some places are still using the global function to get the selected op. Why not use the global function everywhere?

Reply:
my thought, coming from C/Go, is to minimize globals. passing the op around, rather than looking to a global means we can theoretically display/work-with multiple distinct ops at the same time--not now, but in the future. Is this just insanity on my part?